packages/core/template_project/app/build.gradle

```diff
@@ -70,11 +70,14 @@ android {
 
         <% if (webManifestUrl) { %>
```

Comment:
@andreban In what cases is this condition false?

The `webManifestUrl` is now required for Meta Quest too, not only Chrome OS.

Reply:
@andreban What about this?

BTW, in theory, can it be the case that the `chromeosonly` option is passed, but `webManifestUrl` is missing? 🤔

I mean, why is this wrapped in the "if condition" if it is always set. Or is it not always?

Reply:
In the past, that field didn't exist in twa-manifest.json. So, older manifests may not have that information.